CHANGELOG.md

```
@@ -1,11 +1,12 @@
 # Prowide Core - CHANGELOG
 
-#### 10.2.8 - June 2024
-  * Fix: Enhanced the `DefaultMtMetadataStrategy` to prevent NPE when the message headers are malformed
-
-#### 10.2.7 - May 2025
-  * (PW-2055) Fixed the default message metadata extraction for ACK/NAK to set the service message block 1 BIC as receiver, not as sender
-  * (PW-2055) Enhanced the `SwiftMessageUtils` extractors to support the service 21 message type (ACK/NAK)
+#### 10.3.0 - May 2025
```

markdownlint-cli2 (0.17.2)

Heading levels should only increment by one level at a time
Expected: h2; Actual: h4

(MD001, heading-increment)